hc/accounts/tests/test_check_token.py

```diff
@@ -22,7 +22,44 @@ def test_it_redirects(self):
         self.assertEqual(self.profile.token, "")
 
     ### Login and test it redirects already logged in
+    def test_it_redirects_already_logged_in_provided_we_do_a_request_with_the_token_again(self):
```

Comment:
@sir3n-sn this test name could be brief but clear

Reply:
Thanks. Sure ill change them